docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -32,112 +32,229 @@ For example:
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.Web.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop ProductService
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.ProductService.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop ShoppingCartService
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.ShoppingCartService.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+```
 
-  build:
-    name: Build
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      
-      - name: Install Octopus CLI
-        uses: !include <image-version-install-octopus-cli-action>
-        with:
-          version: latest
-          
-      - name: list-octopusdeploy-deployments
-        run: octo list-deployments --server=${{ env.serverURL }} --apiKey=${{ secrets.apiKey }}
+### Creating a release
+Using the Octopus CLI Action, add a step to issue the [create-release](https://octopus.com/docs/octopus-rest-api/octopus-cli/create-release) command to create a release
+
+```yaml
+    - name: Create release
+      run: |
+        octo create-release --project="Octo Pet Shop" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
 ```
 
+### Deploying a release
+To have your GitHub Action build deploy a release, add a step with the [deploy-release](https://octopus.com/docs/octopus-rest-api/octopus-cli/deploy-release) command.
+:::hint
+Use either the `--progress` or `--waitForDeployment` switches to have the build wait for the deployment to complete and report success or failed.  `--progress` will display messages from Octopus itself whereas `--waitForDeployment` will simply wait for Octopus to report success or failure.
+
+```yaml
+    - name: Deploy release
+      run: |
+        octo deploy-release --project="Octo Pet Shop" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}" --releaseNumber="latest" --deployTo="Development"
+```
+:::
+
 :::success
 **Example GitHub Actions Repo:**
 
-View a working GitHub Actions example on our [samples GitHub repository](https://github.com/OctopusSamples/OctopusTrident/blob/master/.github/workflows/packageredgate.yml).
-
-See the corresponding Octopus project on our [samples instance](https://samples.octopus.app/app#/Spaces-106/projects/redgate-feature-branch-example/deployments).
+View a working GitHub Actions examples on our [samples GitHub repository](https://github.com/OctopusSamples).
```

Comment:
We also have the ability to execute a runbook from GH actions.  Perhaps a section on that as well.
https://github.com/marketplace/actions/run-runbook-in-octopus-deploy

Reply:
Added run a runbook section with example YAML.